monkey/infection_monkey/master/automated_master.py

```diff
@@ -198,8 +198,8 @@ def _run_pba(self, pba: Tuple[str, Dict]):
         name = pba[0]
         options = pba[1]
 
-        display_name, command, result = self._puppet.run_pba(name, options)
-        self._telemetry_messenger.send_telemetry(PostBreachTelem(display_name, command, result))
+        display_name, result = self._puppet.run_pba(name, options)
```

Comment:
This is included in PostBreachData too.
```suggestion
        result = self._puppet.run_pba(name, options)
```